pkg/sql/internal_test.go

```diff
@@ -150,6 +150,11 @@ func TestInternalExecAppNameInitialization(t *testing.T) {
 
 	params, _ := tests.CreateTestServerParams()
 	params.Insecure = true
+	params.Knobs.SQLLeaseManager = &sql.LeaseManagerTestingKnobs{
+		// Disable the orphaned leases query which has an innocuous race with the
+		// SHOW QUERIES query used elsewhere in this test.
+		DisableDeleteOrphanedLeases: true,
```

Comment:
you probably want to do the same in `run_control_test.go` since the tests in there are running into the same issue

Reply:
Will do once I fully understand what is going on.